docs/eventing/channels/channel-types-defaults.md

```diff
@@ -5,14 +5,35 @@ type: "docs"
 showlandingtoc: "true"
 ---
 
-Knative provides the InMemoryChannel channel implementation by default. This default implementation is useful for developers who do not want to configure a specific implementation type, such as Apache Kafka or NATSS channels.
+Knative uses two types of channels:
+* The generic Channel object
+* Channel implementations that each have their own custom resource definitions (CRDs), such as
+`InMemoryChannel` and `KafkaChannel`.
 
-**NOTE:** InMemoryChannel channels should not be used in production environments.
+The custom Channel implementations each have their own event delivery mechanisms, such as in-memory
```

Comment:
I think it's a bit confusing to talk about brokers here?

Reply:
I think the idea is to explain that each custom channel implementation is different, and might use different delivery mechanism under the hood.

Reply:
The use of the broker. Especially since we have a broker and Kafka Broker and GCP PubSub broker, so one could be left wondering if Channels must be implemented with our broker model. 
Maybe just say something like:
, such as in-memory or Kafka or GCP Pub/Sub

Reply:
I personally would remove this couple of lines mentioning the broker-based stuff